<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionSelection</class>
 <widget class="QWidget" name="VersionSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #17394A;
background: rgb(240, 240, 240);
</string>
  </property>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>select version</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="versionSelectionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>please select what version you would like</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="versionnextButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color:#17394A;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(210,210,210));
border-radius: 11px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(180,180,180));
}

QPushButton:pressed {
background: qradialgradient(cx: 0.1, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(180,180,180));
}
</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="versionSelectionBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240);
</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>--- Please Select ---</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="useLocalBuildCheckbox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>use a local build on my computer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
